README.md

```diff
@@ -65,7 +65,7 @@ python3 setup.py install
 
 ## Quick Usage
 
-To set up the default folder and file structure for Brownie use:
+To set up the default folder and file structure for Brownie, from your project's directory, use:
```

Comment:
```suggestion
To initialize a new Brownie project, start by creating a new folder. From within that folder, type:
```

Stolen from the documentation. I think this section of the readme hasn't been updated in too long.